test/jdk/java/awt/Focus/UnaccessibleChoice/AccessibleChoiceTest.java

```diff
@@ -42,21 +41,21 @@
 
 import javax.imageio.ImageIO;
 
-/**
+/*
  * @test
  * @bug 4478780
  * @key headful
  * @summary Tests that Choice can be accessed and controlled by keyboard.
  */
+
 public class AccessibleChoiceTest {
     //Declare things used in the test, like buttons and labels here
-    Frame frame = new Frame("window owner");
-    Window win = new Window(frame);
+    Frame frame = new Frame("Accessible Choice Test Frame");
     Choice choice = new Choice();
     Button def = new Button("default owner");
```

Comment:
More meaningful variable name? or may be `button` would also work

```suggestion
    Button defaultOwner = new Button("default owner");
```

Reply:
Done